common/flatpak-transaction.c

```diff
@@ -676,27 +681,38 @@ flatpak_transaction_operation_get_ref (FlatpakTransactionOperation *self)
 }
 
 /**
- * flatpak_transaction_operation_get_related_to_op:
+ * flatpak_transaction_operation_get_related_to_ops:
  * @self: a #FlatpakTransactionOperation
  *
- * Gets the operation which caused this operation to be added to the
- * transaction. In the case of a runtime, it's the app whose runtime it is. In
+ * Gets the operations which caused this operation to be added to the
+ * transaction. In the case of a runtime, it's the apps whose runtime it is (and
+ * this could be multiple apps, if they all require the same runtime). In
  * the case of a related ref such as an extension, it's the main app or
  * runtime. In the case of a main app or something added to the transaction by
- * flatpak_transaction_add_ref(), %NULL will be returned.
+ * flatpak_transaction_add_ref(), %NULL or an empty array will be returned.
  *
  * Note that an op will be returned even if it’s marked as to be skipped when
  * the transaction is run. Check that using
  * flatpak_transaction_operation_get_is_skipped().
  *
```

Comment:
Should we mention that the elements of the array are only safe to access during the lifetime of the transaction? Since we don't take a reference on them.